src/data-visualisation/dashboard-layout/report-pages.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: article
+title: "Layout of a dashboard: Reporting"
+description: "Overview of the report pages to include in NHSBSA reports"
+status: DRAFT
+tags: [data-viz, data-viz-layout]
+order:
+    data-viz: 2
```

Comment:
Same as above ^
Additionally, if this will show up in multiple collections make sure order doesn't conflict with other articles :smile: